src/app/grants/page.tsx

```diff
@@ -0,0 +1,194 @@
+          <span className="text-gray-500">Grant amount</span>
+        </p>
+      </div>
+      <ScrollArea className="mt-4">
+        <div className="flex flex-col gap-4">
+          {grants.map((grant) => (
+            <Card
+              className="border-none shadow-none py-8 px-10"
+              key={grant.title}
+            >
+              <CardContent className="flex items-center gap-8 p-0">
+                <Hexagon className="w-10 h-10" />
+                <div className="flex flex-col gap-2 max-w-2xl">
+                  <p className="font-semibold text-xl">{grant.title}</p>
+                  <p className="text-sm text-gray-500 line-clamp-3">
+                    {grant.description}
+                  </p>
+                  <div className="flex h-5 items-center space-x-4 text-xs text-gray-500 mt-8">
+                    <p>
+                      Date of award:{' '}
+                      <span className="font-semibold text-black">
+                        {grant.date.toLocaleDateString()}
+                      </span>
+                    </p>
+                    <Separator orientation="vertical" />
+                    <div className="flex items-center gap-2">
+                      <p>Delegate to: </p>
+                      <Link
+                        className="group flex items-center font-semibold text-black"
+                        href="/grants"
+                      >
+                        {grant.delegateTo}{' '}
+                        <ArrowUpRight
+                          className="ml-1 text-neutral-500 w-4 h-4 opacity-70 transition-transform duration-300 ease-in-out group-hover:-translate-y-0.5 group-hover:translate-x-0.5 group-hover:opacity-100"
+                          aria-hidden="true"
+                        />
+                      </Link>
+                    </div>
+                    <Separator orientation="vertical" />
+                    <div className="flex items-center gap-2">
+                      <p>Latest claim: </p>
+                      <Link
+                        className="group flex items-center font-semibold text-black"
+                        href="/grants"
+                      >
+                        {grant.latestClaim}{' '}
+                        <ArrowUpRight
+                          className="ml-1 text-neutral-500 w-4 h-4 opacity-70 transition-transform duration-300 ease-in-out group-hover:-translate-y-0.5 group-hover:translate-x-0.5 group-hover:opacity-100"
+                          aria-hidden="true"
+                        />
+                      </Link>
+                    </div>
+                  </div>
+                </div>
+                <div className="flex items-center gap-2 font-semibold">
+                  <span className="text-black">{grant.claimed}</span> /
+                  <span className="text-gray-500">{grant.grantAmount}</span>
+                </div>
+              </CardContent>
+            </Card>
```

Comment:
please extract this into its own component, for example `<GrantCard />`. store it in `components/common`